CONTRIBUTING.md

```diff
@@ -313,61 +313,52 @@ If the related change is essential for the contribution at hand, make clear why
-
-Container system, boot system and library changes are some examples of the pull requests fitting this category.
+If you have enough knowledge and experience in a topic and would like to be a long-term reviewer for related submissions, please contact the current reviewers for that topic.
+The main reviewers for a topic can be hard to find as there is no list, but checking past pull requests or git-blaming the code can give some hints.
 
 ## How to merge pull requests yourself
 [pr-merge]: #how-to-merge-pull-requests
 
-To streamline automated updates, leverage the nixpkgs-merge-bot by simply commenting `@NixOS/nixpkgs-merge-bot merge`.
-The bot will verify if the following conditions are met, refusing to merge otherwise:
+You can use the nixpkgs-merge-bot by commenting `@NixOS/nixpkgs-merge-bot merge`.
+The bot will verify the following conditions, refusing to merge otherwise:
 
 - the PR author should be @r-ryantm or a Nixpkgs committer;
-- the commenter that issued the command should be among the package maintainers;
+- the commenter should be among the package maintainers;
 - the package should reside in `pkgs/by-name`.
 
-Further, nixpkgs-merge-bot will ensure all ofBorg checks (except the Darwin-related ones) are successfully completed before merging the pull request.
-Should the checks still be underway, the bot patiently waits for ofBorg to finish before attempting the merge again.
+Further, nixpkgs-merge-bot will ensure all CI checks and the ofborg builds for Linux have successfully completed before merging the pull request.
+Should the checks still be underway, the bot will wait for them to finish before attempting the merge again.
 
 For other pull requests, please see [I opened a PR, how do I get it merged?](#i-opened-a-pr-how-do-i-get-it-merged).
 
-In case the PR is stuck waiting for the original author to apply a trivial change (a typo, capitalisation change, etc.) and the author allowed the members to modify the PR, consider applying it yourself (or commit the existing review suggestion).
+In case the PR is stuck waiting for the author to apply a trivial change and the author allowed members to modify the PR, consider applying it yourself.
 You should pay extra attention to make sure the addition doesn't go against the idea of the original PR and would not be opposed by the author.
 
-<!--
-The following paragraphs about how to deal with unactive contributors is just a proposition and should be modified to what the community agrees to be the right policy.
-
-Please note that contributors with commit rights unactive for more than three months will have their commit rights revoked.
--->
-
 Please see the discussion in [GitHub nixpkgs issue #321665](https://github.com/NixOS/nixpkgs/issues/321665) for information on how to proceed to be granted this level of access.
```

Comment:
This bit is not really connected to the rest of the text? (maybe out of scope for this PR to fix however)